README.md

```diff
@@ -447,6 +448,181 @@ you can create your own Configuration(s) and compose them with Config's ++ opera
 
 Then you can build as usual with CONFIG=MyConfig.
 
+## <a name="debug"></a> Debugging with GDB
+
+### 1) Generating the RBB emulator
```

Comment:
Can you update this to:

Generating the Remote Bit-Bang (RBB) Emulator

Reply:
Done